lib/interfaces/hapi.js

```diff
@@ -122,8 +122,8 @@ function hapiRegisterFunction(client, config, server, options, next) {
  */
 function handlerSetup(client, config) {
   var hapiPlugin = {register : hapiRegisterFunction.bind(null, client, config)};
-  var version = isPlainObject(config) && isString(config.version)
-                    ? config.version
+  var version = isPlainObject(config) && isString(config.version) ?
+                    config.version
```

Comment:
This is clang formatter automatically formatting this was because of 80col, changing it here would mean a break with other auto-formatted lines